content/docs/integrating-with-other-libraries.md

```diff
@@ -186,21 +186,21 @@ class Chosen extends React.Component {
 }
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/xdgKOz?editors=0010)
+[**CodePen'de Deneyin**](https://codepen.io/gaearon/pen/xdgKOz?editors=0010)
 
 ## Integrating with Other View Libraries {#integrating-with-other-view-libraries}
 
-React can be embedded into other applications thanks to the flexibility of [`ReactDOM.render()`](/docs/react-dom.html#render).
+React, diğer uygulamalara gömülebilir, [`ReactDOM.render()`](/docs/react-dom.html#render)'in esnekliğine teşekkür ediyoruz.
```

Comment:
```suggestion
React, diğer uygulamalara [`ReactDOM.render()`](/docs/react-dom.html#render)'in esnekliği sayesinde gömülebilir.
```

Reply:
In your suggestion, you have dropped the "thanks to". I have read few passages in the Turkish translations of React website. There are similar expressions. I would like to keep the existing one as it is. You are also free if you like to go with your own suggestion.

Reply:
> In your suggestion, you have dropped the "thanks to". I have read few passages in the Turkish translations of React website. There are similar expressions. I would like to keep the existing one as it is. You are also free if you like to go with your own suggestion.

I think the current translation does not express a meaningful sentence. If you want to keep the "thanks to" expression, that's ok for me. But we need to make this sentence natural. I think it's more important to make the sentence understandable.

Reply:
You are online. It might be 12am in Turkey. I was still working the file.

Reply:
Then you have to finish your work before opening the pull request.

Reply:
> Sorry about that. But you should finish your work before opening the pull request.

I did not get the point of this message in this thread. How is it relate to the conversations in the previous messages. 
While I was working on the file, you were messaging.  A pull request has been opened after translation was done. What it supposed to be. Most likely, we do not understand each other. I request you to send this type of messages privately. This time, if you like to say something,  please send them on Slack privately. Thank you.